ci/build-mingw64.sh

```diff
@@ -291,9 +309,11 @@ if [ "$2" = pack ]; then
     dlls=(
         libgcc_*.dll lib{ssp,stdc++,winpthread}-[0-9]*.dll # compiler runtime
         av*.dll sw*.dll lib{ass,freetype,fribidi,harfbuzz,iconv,placebo}-[0-9]*.dll
-        lib{shaderc_shared,spirv-cross-c-shared}.dll zlib1.dll
-        # note: vulkan-1.dll is not here since drivers provide it
+        lib{shaderc_shared,spirv-cross-c-shared}.dll zlib1.dll libdav1d.dll
     )
+    if [[ "$TARGET" != "i686-"* ]]; then
+        dlls+=(vulkan-1.dll)
```

Comment:
problem with including this: (from the build log)
```
  CMake Warning at loader/CMakeLists.txt:216 (message):
    Could not find working MASM assembler
  
    The build will fall back on building with C code
  
    Note that this may be unsafe, as the C code requires tail-call
    optimizations to remove the stack frame for certain calls.  If the compiler
    does not do this, then unknown device extensions will suffer from a
    corrupted stack.
  
 ```

Reply:
The functional test only checks that mpv runs and doesn't crash before printing its version, it doesn't even play a video much less try to use a real VO.
Regardless I think it's likely that we're not even hitting the crash condition in practical use with VK. Also gcc could be compiling the code in a way that it does work.

Still I'd rather not ship a vulkan DLL (which will take precedence over the system one) where its bug-free functioning is not assured.